horace_core/sqw/page_operations/PageOpBase.m

```diff
@@ -1,9 +1,17 @@
 classdef PageOpBase
-    %PAGEOPBASE class defines generic operation, performed on chunk of pixels
+    % PAGEOPBASE class defines generic operation, performed on chunk of pixels
     % located in memory.
     %
-    % An operation normally consists of doing something with pixels and
-    % calculating appropriate avarages to image.
+    % An operation normally consists of doing something with pixels, e.g.
+    % recalculating or modifying signal/variance and
+    % calculating appropriate changes to the image.
+    %
+    % IMPORTANT:
+    % The operation can be used by the algorithms which do not change
+    % the size and shape of the image.
```

Comment:
Do you mean "only"? I would assume that was the simpler case

Reply:
yes, only, added to the description